applications/integrationtests/mechanics/VibrationalModes1D.cpp

```diff
@@ -0,0 +1,99 @@
+#include "BoostUnitTest.h"
+
+#include "mechanics/dofs/DofNumbering.h"
+#include "mechanics/cell/SimpleAssembler.h"
+
+#include "mechanics/integrands/DynamicMomentumBalance.h"
+#include "mechanics/interpolation/InterpolationTrussLobatto.h"
+#include "mechanics/integrationtypes/IntegrationTypeTensorProduct.h"
+#include "mechanics/constraints/Constraints.h"
+#include "mechanics/mesh/UnitMeshFem.h"
+#include "mechanics/mesh/MeshFemDofConvert.h"
+#include "mechanics/tools/CellStorage.h"
+
+#include "mechanics/constitutive/LinearElastic.h"
+
+#include <Eigen/Eigenvalues>
+
+#include <functional>
+
+using namespace NuTo;
+
+struct Steel
+{
+    static constexpr double E = 200.0;
+    static constexpr double nu = 0.3;
+    static constexpr double rho = 8000.;
+};
+
+class VibratingTruss
+{
+public:
+    VibratingTruss(int numElements, int order)
+        : mMesh(UnitMeshFem::CreateLines(numElements))
+        , mOrder(order)
+        , mDof("Dispacement", 1)
+        , mLaw(Steel::E, Steel::nu)
+        , mPDE(mDof, mLaw, Steel::rho)
+        , mIntegrationType(mOrder + 1, eIntegrationMethod::GAUSS)
+    {
+        AddDofInterpolation(&mMesh, mDof, mMesh.CreateInterpolation(InterpolationTrussLobatto(order)));
+
+        mCellGroup = mCells.AddCells(mMesh.ElementsTotal(), mIntegrationType);
+    }
+
+    Eigen::VectorXd Solve()
+    {
+        DofInfo dofInfo = DofNumbering::Build(mMesh.NodesTotal(mDof), mDof, mConstraints);
+        Eigen::SparseMatrix<double> cmat = mConstraints.BuildConstraintMatrix(mDof, dofInfo.numIndependentDofs[mDof]);
+        SimpleAssembler asmbl = SimpleAssembler(dofInfo);
+
+        using namespace std::placeholders;
+
+        auto stiffnessF = std::bind(&Integrands::DynamicMomentumBalance<1>::Hessian0, &mPDE, _1, 0.);
+        auto massF = std::bind(&Integrands::DynamicMomentumBalance<1>::Hessian2, &mPDE, _1);
```

Comment:
Why not just use lambdas here?
~~~cpp
auto stiffnessF = [&](const auto& cipd) { return mPDE.Hessian0(cipd, 0.0); };
auto massF = [&](const auto& cipd) { return mPDE.Hessian2(cipd); };
~~~